WheelPicker.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { requireNativeComponent,View } from 'react-native';
```

Comment:
so you mean it will work for both old and new react version? or is it for some another issue resolver?

Reply:
Basically what I read today `ViewPropTypes` does not exist for production so we you need to use `View.propTypes` for production. So the thing you are suggesting is should be change for this reason.

Reply:
I think it depend on version of React-Native , I've got apple review reject , because  View.propTypes is undefined for RN-0.49.5 with Release configuration, but not sure about it (app is still in apple review)